### PR TITLE
Mention other runtime configuration strategies

### PR DESCRIPTION
**Pull request by stevensonmt**
Might just be me but I really struggled to understand the runtime config setup and feel like this could be useful to others.

### CODE REVIEW

File: getting-started/mix-otp/config-and-releases.markdown [hunk 1]
```diff
@@ -105,6 +105,8 @@ defmodule KV.RouterTest do
 
 Note we removed `async: true` from `use ExUnit.Case`. Since the application environment is a global storage, tests that modify it cannot run concurrently. With all changes in place, all tests should pass, including the distributed one.
 
+Details of other configuration strategies are beyond the scope of this document, but you should be aware that other sources of runtime configuration data can be specified as configuration providers in `mix.exs`. For more details see the documentation for the [Config.Provider module](https://hexdocs.pm/elixir/main/Config.Provider.html#content). Excellent ibraries for many types of config providers can be found on [hex](https://hex.pm) that can ease the use of environment variables and configuration files of various types, such as TOML, YAML, INI, and JSON.
```
Comment (whatyouhide) [inline getting-started/mix-otp/config-and-releases.markdown:108]:
```suggestion
Details of other configuration strategies are beyond the scope of this document, but you should be aware that other sources of runtime configuration data can be specified as configuration providers in `mix.exs`. For more details see the documentation for the [Config.Provider module](https://hexdocs.pm/elixir/main/Config.Provider.html#content). Excellent libraries for many types of config providers can be found on [Hex](https://hex.pm). These can ease the use of environment variables and configuration files of various formats, such as TOML, YAML, INI, and JSON.
```

**Review by whatyouhide — APPROVED** [review]
Nice addition! Thanks @stevensonmt 🙃

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by stevensonmt** [comment]
Wow, you guys are quick. Thanks!

**Pull request merged**